### PR TITLE
Support multiple attachments and custom mimeType

### PR DESCRIPTION
**Pull request by brightsider**

### CODE REVIEW

**Comment by chirag04** [comment]
@brightsider thank you for the PR. this looks good to me. Mind confirming how/where and what was tested here? I won't be able to test it myself so want to make sure this is was tested on ios and android before i merge this.

File: RNMail/RNMail.m [hunk 1]
```diff
@@ -36,102 +36,109 @@ + (BOOL)requiresMainQueueSetup
...
-                };
-                if([supportedMimeTypes objectForKey:attachmentType]) {
-                    mimeType = [supportedMimeTypes objectForKey:attachmentType];
-                } else {
-                    callback(@[[NSString stringWithFormat: @"Mime type '%@' for attachment is not handled", attachmentType]]);
-                    return;
+        if (options[@"attachments"]) {
+            NSArray *attachments = [RCTConvert NSArray:options[@"attachments"]];
+            for (NSDictionary *attachment in attachments) {
+                if (attachment[@"path"] && (attachment[@"type"] || attachment[@"mimeType"])) {
```
Comment (chirag04) [inline RNMail/RNMail.m:73]: can you add `mimeType` to the readme?

Comment (brightsider) [reply RNMail/RNMail.m:73]: Added

**Comment by brightsider** [comment]
@chirag04 yes, it tested on both platforms

**Comment by chirag04** [comment]
Will publish a major version soon

**Comment by systemlevel** [comment]
I'm having an issue with this.  I pulled down and confirmed I'm using the most recent master. The multiple attachments never seem to make it to the gmail email program on Android.  Still need to test on iOS.

**Pull request merged**